env/env_posix.cc

```diff
@@ -789,6 +795,11 @@ class PosixEnv : public Env {
     return thread_pools_[pri].GetBackgroundThreads();
   }
 
+  virtual Status SetAllowNonOwnerAccess(bool allow_non_owner_access) {
```

Comment:
some of the tests are failing because `override` keyword is expected here.

Reply:
added override